package.json

```diff
@@ -52,10 +52,6 @@
   "dependencies": {},
   "devDependencies": {
     "ava": "^0.15.2",
-    "babel": "^6.5.2",
```

Comment:
as mentionned in https://github.com/jscad/csg.js/issues/14, those deps are actually already imported by ava